lib/tasks/check-links/link_checker.rake

```diff
@@ -25,3 +25,11 @@ task "check-links": :environment do
     }
   )
 end
+
+namespace :"check-links" do
+  desc "Check links for a single local authority"
+  task :local_authority, [:authority_slug] => :environment do
+    checker = LocalLinksManager::CheckLinks::LinkStatusRequester.new
```

Comment:
If we add it to a schedule then I would support that, but it'll be used for ad-hoc re-checks at the moment.